browser/src/index.tsx

```diff
@@ -251,6 +251,10 @@ const start = (args: string[]) => {
             return
         }
 
+        if (key === "|") {
+            pluginManager.findAllReferences()
+        }
```

Comment:
Good question, I actually just picked something quick and easy for testing. 

Do you have any ideas for a better default key for this? Another option would be to have it exposed in the app menu.